lib/utils/watchers.js

```diff
@@ -148,17 +149,21 @@ function handleRedditEvent({ target, detail: { type, data } }) {
 		(target.expando._.update || 0) + 1 :
 		0;
 
-	data._ = {
```

Comment:
Firefox throws on assigning cross-domain properties to an object (data from an object from reddit.com). The complement to Firefox throwing on Reddit code trying to get values from an extension's event.